pickee_frontend/src/assets/styles/common/typography.css

```diff
@@ -0,0 +1,68 @@
+:root {
+    --font-family: "Montserrat", sans-serif;
+
+    --font-size-general: 24px;
+    --font-line-height-general: 29px;
+    --font-weight-general: 300;
+
+    --font-size-small: 18px;
```

Comment:
I suggest using the Material Design [font size](https://material.io/design/typography/the-type-system.html#type-scale) naming convention. It provides more semantics and scales nicer (no `font-line-height-medium`).

Reply:
I’m going to look what options are provided in vuetify